db/migrate/20160816144624_create_damages.rb

```diff
@@ -0,0 +1,14 @@
+class CreateDamages < ActiveRecord::Migration
+  def change
+    create_table :damages do |t|
+      t.string :uuid
+      t.string :damage_issued_reservation_uuid
```

Comment:
So do you think I should make any change of these two uuids in the damage table?